src/components/bridge-info-state.ts

```diff
@@ -140,4 +206,12 @@ export class BridgeInfoStateSyncer<BridgeMappingInfo> {
         }
         return content;
     }
+
+    public createEventFeaturesContent(mapping: MappingInfo)
+    : MSC3968Content|null {
+        if (!mapping.eventFeatures) {
+            return null;
+        }
+        return mapping.eventFeatures;
+    }
```

Comment:
Create implies this isn't atomic, when really we're just returning some info. May I offer you a getter.

```suggestion
    public get eventFeaturesContent(mapping: MappingInfo)
    : MSC3968Content|null {
        return mapping?.eventFeatures ?? null;
    }
```

Reply:
Aren't getters supposed to take no parameters?

Reply:
It would be so helpful if I could read.

Reply:
Sorry for the delay applying this, I didn't understand what you meant by "Have removed a space." and didn't notice the edit